lib/liblinear.rb

```diff
@@ -26,6 +26,8 @@ class Liblinear
   L2R_L2LOSS_SVR_DUAL = Liblinearswig::L2R_L2LOSS_SVR_DUAL
   L2R_L1LOSS_SVR_DUAL = Liblinearswig::L2R_L1LOSS_SVR_DUAL
 
+  LIBLINEAR_VERSION   = sprintf("%.2f", Liblinearswig::LIBLINEAR_VERSION / 100.0)
+
```

Comment:
Why not the integer version number? `Liblinear::VERSION` is also a string, and this is forward-compatible with non `x.yy` version numbers if they would be occuring in the future.